### ISSUE TITLE
Question about installing msix package 

### ISSUE THREAD

**Issue by DevX-Cipher** [WinMerge/winmerge#2340]
I am interested in understanding the process you are using to install the application and the .msix package, specifically regarding the use of external locations during the setup. Could you please provide an explanation of how this is being managed?

Thank you.

**Comment by sdottaka** [comment]
I don't know if it's the proper way, but in WinMerge I use the Add-AppxPackage command in Powershell to install msix files (Explorer context menu for Windows 11). See below:

https://github.com/WinMerge/winmerge/blob/f2f83a8465bb57cbfaee176150af04309646983d/Installer/InnoSetup/WinMergeX64.iss#L768

**Comment by modz2014** [comment]
yes i tried this in a test but it fails because you cant install external msix package without installing it externally

**Comment by sdottaka** [comment]
The msix package of WinMerge ShellExtension is a type of package called a Sparse Package.
The following URL may be helpful:
https://blogs.windows.com/windowsdeveloper/2019/10/29/identity-registration-and-activation-of-non-packaged-win32-apps/

**Comment by modz2014** [comment]
yes I know it but you can't install them unless externally if you read your manifest file it has external at the top another words if I try to open the msix package it wont install

**Comment by modz2014** [comment]
@sdottaka there is still context menu showing up twice once on the new windows 11 context menu and the one under show more options

**Comment by sdottaka** [comment]
I may have misunderstood what you’re saying, but the WinMerge shell extension for Windows 11 is expected to appear in both the right-click menu in Explorer and the menu that appears when you click "Show more options."

Or are you concerned about multiple WinMerge menu items appearing in the "Show more options" menu? If so, please click the "Unregister shell extension" button to unregister the shell extension for Windows 10 and earlier.

![image](https://github.com/WinMerge/winmerge/assets/98126/b8fadefe-ee04-40dd-b759-82e60018dda7)

**Comment by modz2014** [comment]
it show on the new windows 11 context menu and under show more options

**Comment by sdottaka** [comment]
Could you please clarify what your expected behavior is? Additionally, could you attach a screenshot of the current display?

**Comment by modz2014** [comment]
here it shows the new Windows 11 context menu and also under show more options

![Contextmenu](https://github.com/WinMerge/winmerge/assets/25472227/1eccf737-0285-4c3d-8d51-100a86e630f0)
![Contextmenu1](https://github.com/WinMerge/winmerge/assets/25472227/bfea7a6d-d972-460e-af28-2a284744cb14)

**Comment by sdottaka** [comment]
The display in the screenshot above is as we expected.

Generally, ShellExtension created to be displayed in the right-click menu of Windows 11 Explorer will also be displayed in the Show more options menu. I think it may be difficult or require special ingenuity to display it in only one of them.

**Comment by modz2014** [comment]
is it possible of something you can do please

**Comment by sdottaka** [comment]
Do you expect the behavior to be that the WinMerge menu does not appear in the Show more options menu?

So far, I have not found a way to change that behavior, because ShellExtension does not care which side it is called from.

**Comment by modz2014** [comment]
maybe under Get state of shared counter I was looking into how notepad++ does it have looked into this
